.github/workflows/tests.yaml

```diff
@@ -601,6 +601,54 @@ jobs:
                      --tool-args "onnetwork-long --nodeid 1 --setup-pin-code 20202021 --discriminator 3840 -t 1000" \
                      --factoryreset \
                   '
+              - name: Run IM Invoke Test
+              timeout-minutes: 10
```

Comment:
Could we change the timeout to 1, 10 mins is too long for a IM command.

Reply:
offline chat, it has pairing test inside, so we align the timeout 10min with the previous pairing test.